README.md

```diff
@@ -7,31 +7,45 @@ The steps outlined below are based on the assumption that you are deploying the
 
 ### Install and Configure Dependencies
 ```bash
+# (required) installing lxd and juju
 sudo snap install lxd --classic
 sudo snap install juju --classic
+
+# (optional) installing terraform
+sudo apt install gpg
+wget -O- https://apt.releases.hashicorp.com/gpg | sudo gpg --dearmor -o /usr/share/keyrings/hashicorp-archive-keyring.gpg
+echo "deb [arch=$(dpkg --print-architecture) signed-by=/usr/share/keyrings/hashicorp-archive-keyring.gpg] https://apt.releases.hashicorp.com $(lsb_release -cs) main" | sudo tee /etc/apt/sources.list.d/hashicorp.list
+sudo apt update
+sudo apt install terraform
```

Comment:
**question:** Am not an Ubuntu user, tested this in a VM but if anyone can check this that would be great.

Reply:
I'd replace this with `sudo snap install terraform --classic` :)